blenderproc/python/loader/URDFLoader.py

```diff
@@ -0,0 +1,454 @@
+            mat = bpy.data.materials[viscol_tree.material.name]
+            mat = Material(mat)
+        else:
+            # create a new material
+            mat = MaterialLoaderUtility.create(name=viscol_tree.material.name)
+            # create a principled node and set the default color
+            principled_node = mat.get_the_one_node_with_type("BsdfPrincipled")
+            color = viscol_tree.material.color
+            if color is None:
+                color = Vector([1., 1., 1., 1.])
+            principled_node.inputs["Base Color"].default_value = color
+        obj.replace_materials(mat)
+
+        # check for textures
+        if viscol_tree.material.texture is not None:
+            # image should've been loaded automatically
+            mat = MaterialLoaderUtility.create(name=viscol_tree.material.name + "_texture")
+            nodes = mat.nodes
+            links = mat.links
+
+            color_image = nodes.new('ShaderNodeTexImage')
+
+            if not os.path.exists(viscol_tree.material.texture.filename):
+                raise Exception(f"Couldn't load texture image for {viscol_tree} from "
+                                f"{viscol_tree.material.texture.filename}")
+            color_image.image = bpy.data.images.load(viscol_tree.material.texture.filename, check_existing=True)
+
+            principled = Utility.get_the_one_node_with_type(nodes, "BsdfPrincipled")
+            links.new(color_image.outputs["Color"], principled.inputs["Base Color"])
+
+            obj.replace_materials(mat)
+
+    # set the pose of the object
+    try:
+        obj.set_local2world_mat(Matrix(viscol_tree.origin))
+        print(f"Set matrix_local of {obj.get_name()} to \n{viscol_tree.origin}")
+    except Exception as e:
+        print(f"No origin found for {viscol_tree}: {e}. Setting origin to world origin.")
+        obj.set_local2world_mat(Matrix.Identity(4))
+
+    # set scale of the mesh
+    scale = get_size_from_geometry(viscol_tree.geometry)
+    if scale is not None:
+        obj.set_scale([scale, scale, scale])
+        obj.persist_transformation_into_mesh(location=False, rotation=False, scale=True)
+
+    return obj
+
+
+def load_inertial(inertial_tree: "urdfpy.Inertial", name: str) -> Inertial:
+    """ Loads an inertial element from an urdf tree.
+
+    :param inertial_tree: The urdf representation of the inertial element.
+    :param name: Name if the inertial element.
+    :return: The respective Inertial object.
+    """
+    # create new primitive
+    primitive = create_primitive(shape="CUBE")
+    inertial = Inertial(primitive.blender_obj)
+    inertial.set_name(name=name)
+
+    # set inertial-specific attributes
+    inertial.set_origin(origin=inertial_tree.origin)
+    inertial.set_mass(mass=inertial_tree.mass)
+    inertial.set_inertia(inertia=inertial_tree.inertia)
+    primitive.blender_obj.dimensions = Vector([0.03, 0.03, 0.03])  # just small values to keep cubes small for debugging
```

Comment:
Shouldn't this be a default value in the default_values.py?

Reply:
I don't know, should it?